lib/GADS.pm

```diff
@@ -1329,6 +1329,49 @@ any ['get', 'post'] => '/settings/team_edit/:id' => require_any_role [qw/useradm
     };
 };
 
+any [ 'get', 'post' ] => '/settings/report_defaults/' => require_role 'superadmin'=> sub {
+    my $site             = var 'site';
+
+    if(body_parameters->get('submit')) {
+        my $post_marking = body_parameters->get('security_marking');
+
+        my $txn_scope_guard = schema->txn_scope_guard;
+
+        $site->update({ security_marking => $post_marking });
+
+        $txn_scope_guard->commit;
```

Comment:
Doesn't really matter, but just to note that there is no need for a guard if only making one SQL change